diff --git a/CMakeLists.txt b/CMakeLists.txt
index fc9a268e..e8304fe2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -150,10 +150,8 @@ endif()
 # For miscellaneous files
 if(SFML_OS_WINDOWS OR SFML_OS_IOS)
     set(DEFAULT_INSTALL_MISC_DIR .)
-elseif(SFML_OS_LINUX OR SFML_OS_FREEBSD OR SFML_OS_OPENBSD OR SFML_OS_NETBSD)
+elseif(SFML_OS_LINUX OR SFML_OS_FREEBSD OR SFML_OS_OPENBSD OR SFML_OS_NETBSD OR SFML_OS_MACOSX)
     set(DEFAULT_INSTALL_MISC_DIR share/SFML)
-elseif(SFML_OS_MACOSX)
-    set(DEFAULT_INSTALL_MISC_DIR /usr/local/share/SFML)
 elseif(SFML_OS_ANDROID)
     set(DEFAULT_INSTALL_MISC_DIR ${CMAKE_ANDROID_NDK}/sources/third_party/sfml)
 endif()
